book/src/proposals/durable-tx-nonces.md

```diff
@@ -0,0 +1,49 @@
+# Durable Transaction Nonces
+
+## Problem
+
+To prevent replay, Solana transactions contain a nonce field populated with a "recent" blockhash value. A transaction containing a blockhash that is too old (~2min as of this writing) is rejected by the network as invalid. Unfortunately certain use cases, such as custodial services, require a longer time allowance to produce a signature for the transaction. A mechanism is needed to enable these slow-signer network participants.
+
+## A Naive Solution
+
+An initial attempt at this was made by designating an special blockhash value that, if present in the `recent_blockhash` field, allowed the transaction to succeed so long as it additionally spent the full balance of the fee account.  However, this would introduce a security vulnerability by the fact that the transaction could be replayed by simply funding the fee account again.
+
+## A Contract-based Solution
+
+Here we propose a contract-based solution to the problem, whereby a client can "stash" a nonce value for future use in a transaction's `recent_blockhash` field. This approach is akin to the Compare and Swap atomic instruction, implemented by some CPU ISAs.
```

Comment:
I find that if you write this as if it's already implemented, you'll have less trouble moving it to "implemented".

All of the review can happen in the PR, and rejected proposals can be put in an "others considered" section.

See rent ;)